src/gui/screen_reset_error.cpp

```diff
@@ -0,0 +1,31 @@
+// screen_reset_error.cpp
+
+#include "screen_reset_error.hpp"
+#include "config.h"
+#include "ScreenHandler.hpp"
+#include "sound.hpp"
+
+screen_reset_error_data_t::screen_reset_error_data_t()
+    : window_frame_t() {
+
+    ClrMenuTimeoutClose();
+    ClrOnSerialClose();
+    SetBackColor(COLOR_RED);
+}
+
+void screen_reset_error_data_t::draw() {
+    window_frame_t::draw();
+    /// avoid collision of sounds
+    Sound_Stop();
```

Comment:
Are you sure that this method is not called periodically so that this sound is staring to play every time ?
CriticalAlert is repeating sound so integrity of this sound could be compromised.